### ISSUE TITLE
Chrome about 20% faster than Brave

### ISSUE THREAD

**Issue by wooque** [brave/brave-browser#20891]
## Description 
Run Speedometer benchmark (https://browserbench.org/Speedometer2.0/) on Chrome with uBlock Origin against Brave to test raw performance.

Brave: 105
Chrome with uBlock Origin: **125**
Chrome: 138

So it turns out Chrome is about **20%** faster in raw performance. 
Without uBlock Origin it's even faster, but with uBlock Origin is more apple-to-apple comparison.
Tested on Linux (Debian 11).

## Steps to Reproduce

   1. Install latest Chrome and Brave. 
   2. Install uBlock Origin on Chrome.
   3. Run Speedometer - https://browserbench.org/Speedometer2.0/
   4. Observe performance difference

## Actual result:

Chrome with uBlock Origin about 20% faster than Brave.

## Expected result:
Chrome with uBlock Origin and Brave should be on the same level of performance.

## Reproduces how often: 

Easily reproduced

## Brave version (brave://version info)

Brave | 1.35.100 Chromium: 98.0.4758.87 (Official Build) (64-bit)
-- | --
Revision | e4cd00f135fb4d8edc64c8aa6ecbe7cc79ebb3b2-refs/branch-heads/4758@{#1002}
OS | Linux

## Version/Channel Information:

- Can you reproduce this issue with the current release? Yes
- Can you reproduce this issue with the beta channel? N/A
- Can you reproduce this issue with the nightly channel? N/A

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? No.
- Does the issue resolve itself when disabling Brave Rewards? No.
- Is the issue reproducible on the latest version of Chrome? No.

**Comment by stephendonner** [comment]
@wooque yup, thanks for the report.  We're working on it, starting with PGO, over in https://github.com/brave/brave-browser/issues/20086.
